instance/tests/test_github.py

```diff
@@ -93,7 +93,7 @@ def test_get_pr_by_number(self):
             '**Description**\r\n\r\nHello!\nDesc with unicode «ταБЬℓσ»\r\n'
             '- - -\r\n**Settings**\r\n```yaml\r\nEDXAPP_FEATURES:\r\n  ALLOW: true\r\n```')
         self.assertEqual(pr.number, 8474)
-        self.assertEqual(pr.fork_name, 'open-craft/edx-platform')
+        self.assertEqual(pr.fork_name, 'edx/edx-platform')
```

Comment:
@Kelketek Yes, it will fix the PR URL, but this will break the repository URL, no?

https://github.com/open-craft/opencraft/blob/master/instance/templates/instance/ansible/vars.yml#L42
https://github.com/open-craft/opencraft/blob/master/instance/models/instance.py#L329
https://github.com/open-craft/opencraft/blob/master/instance/models/instance.py#L310

Actually, given that this can be pretty confusing to keep in mind, it might be worth adding a test specifically about that.